package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-simple-auth-token",
-  "version": "4.0.7",
+  "version": "4.0.8",
```

Comment:
Dropping support for node 6 is a breaking change and requires a major version release. But I think versions are not updated as part of pull requests at all, are they?

Reply:
Good point. Node v6 is now still supported. There is no need for this library to force a specific version.